Makefile

```diff
@@ -69,6 +69,15 @@ docker_test_lint:
 		$(REGISTRY_URL)/${DOCKER_IMAGE_DEVELOPER_TOOLS}:${DOCKER_TAG_VERSION_DEVELOPER_TOOLS} \
 		/usr/local/bin/test_lint.sh
 
+# Execute lint tests in github actions
+.PHONY: docker_test_lint_gha
+docker_test_lint_gha:
+	docker run --rm \
+		-v $(CURDIR):/workspace \
+		$(REGISTRY_URL)/${DOCKER_IMAGE_DEVELOPER_TOOLS}:${DOCKER_TAG_VERSION_DEVELOPER_TOOLS} \
+		/usr/local/bin/test_lint.sh --markdown --contrib-guide=https://github.com/terraform-google-modules/terraform-google-cloud-storage/blob/master/CONTRIBUTING.md
```

Comment:
Can we make this a relative path?

Reply:
So I think relative paths for PRs are still buggy: https://github.com/github/markup/issues/576
I was able to do a workaround. PTAL

Reply:
Example with relative path staged here: https://github.com/bharathkkb/terraform-google-cloud-storage/pull/8

Reply:
Great, let's also make that the default (in the central script) so we don't have to specify it for each repo.